src/Components/Web.JS/src/GlobalExports.ts

```diff
@@ -12,7 +12,7 @@ import { NavigationLock } from './NavigationLock';
 import { DefaultReconnectionHandler } from './Platform/Circuits/DefaultReconnectionHandler';
 import { CircuitStartOptions } from './Platform/Circuits/CircuitStartOptions';
 import { WebAssemblyStartOptions } from './Platform/WebAssemblyStartOptions';
-import { Platform, Pointer, System_String, System_Array, System_Object, System_Boolean, System_Byte, System_Int } from './Platform/Platform';
+import { Platform, Pointer, System_String, System_Array, System_Boolean, System_Byte, System_Int } from './Platform/Platform';
```

Comment:
is something still using `System_String`, `System_Array` etc ? Those are mono naked pointers.